src/video-player.const.js

```diff
@@ -50,7 +51,8 @@ export const CLOUDINARY_CONFIG_PARAM = [
   'url_suffix',
   'use_root_path',
   'auth_token'
-];
+  // Support camelCase as well
+].map((param) => ([param, camelize(param)])).flat();
```

Comment:
see my first comment since it can remove the need for doing this, but anyway i personally would not do it programmatically but would just introduce it manually, more code but easier to read IMO and more safe.